File: app/src/main/java/ru/argerd/repo/searchScreen/PagerAdapter.java
package ru.argerd.repo.searchScreen;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

public class PagerAdapter extends FragmentPagerAdapter {
    private String[] tabTitles = {"По мероприятиям", "По НКО"};

    PagerAdapter(@NonNull FragmentManager fm, int behavior) {
        super(fm, behavior);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return PageFragment.newInstance((byte) 0);
            case 1:
            default:
                return PageFragment.newInstance((byte) 1);
        }
    }

    @Override
    public int getCount() {
        return 2;
    }

    @Nullable
    @Override
    public CharSequence getPageTitle(int position) {
        return tabTitles[position];
    }
}

File: app/src/main/java/ru/argerd/repo/adapters/PagerAdapter.kt
package ru.argerd.repo.adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import ru.argerd.repo.screens.search.page.PageFragment

class PagerAdapter(fm: FragmentManager, behavior: Int) : FragmentPagerAdapter(fm, behavior) {
    private val tabTitles = arrayOf("По мероприятиям", "По НКО")

    override fun getItem(position: Int): Fragment {
        return when (position) {
            0 -> PageFragment.newInstance(0.toByte())
            1 -> PageFragment.newInstance(1.toByte())
            else -> PageFragment.newInstance(1.toByte())
        }
    }

    override fun getCount(): Int {
        return 2
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return tabTitles[position]
    }
}